Ombi.Services/Jobs/RecentlyAdded.cs

```diff
@@ -200,16 +200,17 @@ private void Start(NewletterSettings newletterSettings, bool testEmail = false)
                 html = template.LoadTemplate(sb.ToString());
                 Log.Debug("Loaded the template");
             }
-            
 
 
-            Send(newletterSettings, html, plexSettings, testEmail);
+            string escapedHtml = new string(html.Where(c => !char.IsControl(c)).ToArray());
```

Comment:
I plan on adding a feature on the Newsletter tab allowing you to email everyone on your Plex updates for downtime and what not with a custom message section. Also do you have a backlog of features? I'm looking for a new project to start committing to and get my C# chops back up and build my Github profile out a bit :-)